pallets/registrar/src/lib.rs

```diff
@@ -1182,3 +1343,22 @@ where
         Ok(frame_system::RawOrigin::Signed(manager).into())
     }
 }
+
+// TODO: import this from dancekit
+pub const REGISTRAR_PARAS_INDEX: &[u8] =
+    &hex_literal::hex!["3fba98689ebed1138735e0e7a5a790abcd710b30bd2eab0352ddcc26417aa194"];
+
+// Need to copy ParaInfo from
+// polkadot-sdk/polkadot/runtime/common/src/paras_registrar/mod.rs
+// Because its fields are not public...
+// TODO: import this from dancekit
```

Comment:
Let's make a PR to parity to make them public, it does not make any sense that they are not